knowledge-base/kb-carousel-howto-get-topmost-item.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Get the topmost item in RadCarousel
+description: How to obtain the top CarouselItem
+type: how-to
+page_title: Find the top item in RadCarousel
```

Comment:
Can we use "element" instead of "item" here? Just to include as many relevant words as possible.